fabric_cf/orchestrator/core/orchestrator_slice_wrapper.py

```diff
@@ -163,6 +227,117 @@ def create(self, *, bqm_graph: Neo4jCBMGraph, slice_graph: Neo4jASM,
 
     @staticmethod
     def __validate_node_sliver(sliver: NodeSliver):
+        """
+        Validate Network Node Sliver
+        @param sliver Node Sliver
+        @raises exception for invalid slivers
+        """
         if sliver.get_capacities() is None and sliver.get_capacity_hints() is None:
             raise OrchestratorException(message="Either Capacity or Capacity Hints must be specified!",
                                         http_error_code=OrchestratorException.HTTP_BAD_REQUEST)
+
+    def __build_network_service_reservations(self, slice_graph: Neo4jASM,
+                                             node_res_mapping: Dict[str, str]) -> List[TicketReservationAvro]:
+        """
+        Build Network Service Reservations
+        @param slice_graph Slice graph
+        @param node_res_mapping Mapping of Network Node sliver Id to reservation Id;
+                                Needed to add dependency information in network service slivers
+        @return list of network service reservations
+        """
+        reservations = []
+        for ns_id in slice_graph.get_all_network_service_nodes():
+
+            # Build Network Service Sliver
+            sliver = slice_graph.build_deep_ns_sliver(node_id=ns_id)
+            sliver_type = sliver.get_type()
+
+            # Ignore Sliver types P4,OVS and MPLS
+            if sliver_type == ServiceType.P4 or sliver_type == ServiceType.OVS or sliver_type == ServiceType.MPLS:
```

Comment:
Maybe have a class member list of 'ignorable' services - it may grow/change over time